src/governance-object.h

```diff
@@ -25,6 +25,7 @@ class CGovernanceVote;
 
 static const int MAX_GOVERNANCE_OBJECT_DATA_SIZE = 16 * 1024;
 static const int MIN_GOVERNANCE_PEER_PROTO_VERSION = 70204;
+static const int GOVERNANCE_FILTER_PROTO_VERSION = 70206;
```

Comment:
I think it's better to keep all governance parameters in the same file.  They're in this one to avoid a circular header dependency.